### DISCUSSION TITLE
xrepo支持本地切换镜像站？

### DISCUSSION THREAD

**Discussion by Extra-Creativity** [xmake-io/xmake#4462] [Ideas]
github在国内有一些镜像站（比如[hub.nuaa.cf](https://hub.nuaa.cf/)），xrepo应该很多包都是只有github一个源吧，有没有什么选项在本地设置镜像源，能在github联不通的时候切换（就像pip换清华源一样）？

**Comment by waruqi** [comment]
https://xmake.io/#/zh-cn/package/remote_package?id=%e9%95%9c%e5%83%8f%e4%bb%a3%e7%90%86

**Reply by Extra-Creativity** [reply]
我愚蠢了，问之前搜了镜像但没结果，可能是我忘切换中文了XD
